client/src/component/Review.jsx

```diff
@@ -1,29 +1,98 @@
 import React from 'react';
+import styles from '../componentsStyles/ReviewStyle.jsx';
+
 
 class Review extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      reviewsHaveBeenClicked: false,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event, review) {
+    event.preventDefault();
+    review.reviewClicked = true;
+    this.setState({ reviewsHaveBeenClicked: true });
   }
 
   render() {
+    const {
+      UserProfilePicture,
+      Username,
+      Time,
+      ReviewContainer,
+      ReadMore,
+      TimeUsernameProfilePicContainer,
+      TimeAndUsernameContainer,
+      Message,
+    } = styles;
+    
+    const { reviewsHaveBeenClicked } = this.state;
     const { review } = this.props;
     const {
-       profile_pic_url, username, time_made, message 
+      profile_pic_url, 
+      username,
+      time_made,
+      message,
+      reviewClicked,
     } = review;
-    return (
-      <div>
-        <div>
+    const testMessage = message.slice(0, 600);
+    console.log(testMessage);
+    if (reviewsHaveBeenClicked && reviewClicked) {
+      return (
```

Comment:
I don't know if its possible but try to refactor with styled component to have one return statement at the end... if it's tricky or not possible, I would not worry about it.